test/train-sets/ref/ccb_quad_save_resume.stderr

```diff
@@ -8,17 +8,17 @@ power_t = 0.5
 using no cache
 Reading datafile = train-sets/ccb_test_interactions.dat
 num sources = 1
-Enabled reductions: gd, scorer, csoaa_ldf, cb_adf, cb_explore_adf_greedy, cb_sample, shared_feature_merger, ccb_explore_adf
+Enabled reductions: gd, generate_interactions, scorer, csoaa_ldf, cb_adf, cb_explore_adf_greedy, cb_sample, shared_feature_merger, ccb_explore_adf
 average  since         example        example  current  current  current
 loss     last          counter         weight    label  predict features
-0.000000 0.000000            1            1.0 0:0,1:0,... 0,1,2,...      135
-0.000000 0.000000            2            2.0 3:0,4:0,... 3,4,0,...       89
-0.000000 0.000000            4            4.0 1:0,4:-1,... 1,4,2,...       89
--0.285714 -0.571429            8            8.0 4:-1,1:0,... 4,1,0,...       89
+0.000000 0.000000            1            1.0 0:0,1:0,... 0,1,2,...      199
```

Comment:
Any idea why the number of features actually drops here? Mostly just curious about this one..

Reply:
The first example vs the second one here? It's just the fact that the first input example has an extra shared feature